Reland "[GLIB] REGRESSION(250811@main): Undefined subroutine &Test262::Runner::isWindows"

emw-apple opened the pull request:
#### 86adbbfd98ab2205c02b56e074687e3c56895d4f
```
Reland "[GLIB] REGRESSION(250811@main): Undefined subroutine &Test262::Runner::isWindows"
https://bugs.webkit.org/show_bug.cgi?id=243454

Reviewed by Ross Kirsling and Stephanie Lewis.

This reverts commit aff5e1addc0ed7ff4198f0407e692bbddc70d094.

Add additional exported functions to webkitdirs. Add perl module imports
where needed in Tools/ scripts, as identified by running `perl
-MO=Lint,none,undefined-subs`.

* Tools/Scripts/make-new-script-test:
* Tools/Scripts/run-javascriptcore-tests:
* Tools/Scripts/test262/Runner.pm:
* Tools/Scripts/update-iexploder-cssproperties:
* Tools/Scripts/update-webkit-libs-jhbuild:
* Tools/Scripts/webkitdirs.pm:
* Tools/Scripts/webkitperl/FeatureList.pm:

Canonical link: https://commits.webkit.org/253165@main
```

rkirsling reviewed (approved):
Thanks for all of this. Perl sure can be a nightmare... 😰

emw-apple committed:
Reland "[GLIB] REGRESSION(250811@main): Undefined subroutine &Test262::Runner::isWindows"
https://bugs.webkit.org/show_bug.cgi?id=243454

Reviewed by Ross Kirsling and Stephanie Lewis.

This reverts commit aff5e1addc0ed7ff4198f0407e692bbddc70d094.

Add additional exported functions to webkitdirs. Add perl module imports
where needed in Tools/ scripts, as identified by running `perl
-MO=Lint,none,undefined-subs`.

* Tools/Scripts/make-new-script-test:
* Tools/Scripts/run-javascriptcore-tests:
* Tools/Scripts/test262/Runner.pm:
* Tools/Scripts/update-iexploder-cssproperties:
* Tools/Scripts/update-webkit-libs-jhbuild:
* Tools/Scripts/webkitdirs.pm:
* Tools/Scripts/webkitperl/FeatureList.pm:

Canonical link: https://commits.webkit.org/253165@main
